README.md

```diff
@@ -3,7 +3,7 @@
 This project is for sending slack/email/SQS messages in response to events. It supports several types of event:
 
 * ECR scan results. Each time there is an ECR repository scan, the scan results are checked. If there are any errors, a
-  Slack and email message is sent.
+  Slack.
```

Comment:
Probably should reword as _Slack message is sent._

Reply:
Good spot!